custom-types/react-augment.d.ts

```diff
@@ -1,12 +1,13 @@
 // https://stackoverflow.com/questions/58469229/react-with-typescript-generics-while-using-react-forwardref/58473012
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
the import is needed to expose existing React modules, so have ignored the error